nextcord/appinfo.py

```diff
@@ -72,11 +73,6 @@ class AppInfo:
         grant flow to join.
     rpc_origins: Optional[List[:class:`str`]]
         A list of RPC origin URLs, if RPC is enabled.
-    summary: :class:`str`
```

Comment:
i would highly recommend not making this a breaking change and instead setting to an empty string for backwards compatibility

Reply:
But v2 is trying to keep at least some backwards compatibility currently as we are trying to make it stable

maybe instead of unsure changes we can just work on what's confirmed now